src/cgame/cg_tutorial.cpp

```diff
@@ -351,29 +351,17 @@ static void CG_HumanText( char *text, playerState_t *ps )
 {
 	if ( !ps->ammo && !ps->clips && !BG_Weapon( ps->weapon )->infiniteAmmo )
 	{
-		//no ammo
-		switch ( ps->weapon )
+		// Need to resupply ammo
```

Comment:
How dare you remove one of those beautifully unmaintainable switches by a sane check? /s